docs/news.rst

```diff
@@ -0,0 +1,104 @@
+News Workflow
+=============
+
+One of the most helpful features of rever is the ``changelog`` activity.
+This activity produces a changelog by colating ``news`` files.
+The changelog is written into the repo and can be used in the GitHub
+release activity.
+
+The workflow for using news is:
+  1. When changes are added to the code (usually via a Pull Request) add a 
+     file to the ``news`` directory. 
+     Usually we name the file after the branch that is used to avoid conflicts. 
+     The news files have two formats depending on when the rever template
+     was made.
+     The older templates look like::
+
+       **Added:** None
+
+       **Changed:** None
+
+       **Deprecated:** None
+
+       **Removed:** None
+
+       **Fixed:** None
+
+       **Security:** None
+     
+     To add a news entry, remove the ``None`` from the category, add a
+     newline and ``* My news entry``.
+     For example::
+
+       **Added:**
+
+       * News tutorial
+
+       **Changed:** None
+
+       **Deprecated:** None
+
+       **Removed:** None
+
+       **Fixed:** None
+
+       **Security:** None
```

Comment:
I think we should call them style 1 & 2, since we may not deprecate them and we may add other styles. I'd also switch the order since we want people to use the "new" style.